x-pack/platform/plugins/shared/alerting_v2/server/config.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import type { TypeOf } from '@kbn/config-schema';
+
+export const configSchema = schema.object({
+  esqlRules: schema.object({
+    enabled: schema.boolean({ defaultValue: true }),
+  }),
+});
```

Comment:
```suggestion
export const configSchema = schema.object({
  enabled: schema.boolean({ defaultValue: true }),
});
```

Unless we need to disable specific features of the new alerting engine maybe we can simplify and either have the plugin enabled or disabled. Either you get it all or not. I am also wondering if we need a config as we work on a feature branch at the moment.